ci-conda.Dockerfile

```diff
@@ -1,12 +1,21 @@
 ARG CUDA_VER=notset
 ARG LINUX_VER=notset
 ARG PYTHON_VER=notset
+ARG YQ_VER
+ARG AWS_CLI_VER
+
+FROM mikefarah/yq:${YQ_VER} as yq
+
+FROM amazon/aws-cli:${AWS_CLI_VER} as aws-cli
+
 FROM rapidsai/miniforge-cuda:cuda${CUDA_VER}-base-${LINUX_VER}-py${PYTHON_VER}
 
 ARG TARGETPLATFORM
 ARG CUDA_VER
 ARG LINUX_VER
 ARG PYTHON_VER
+ARG CODECOV_VER
+ARG SCCACHE_VER
```

Comment:
In the `ci-wheel.Dockerfile` file, the `ARG` statements are placed right above the `RUN` statement where they are used.

This file is inconsistent with that file.

Let's move these `ARG`s above the `RUN` statements for consistency with `ci-wheel.Dockerfile`.